src/NexusMods.DataModel/Cursors.cs

```diff
@@ -0,0 +1,11 @@
+﻿using NexusMods.DataModel.Loadouts;
+
+namespace NexusMods.DataModel;
+
+/// <summary>
+/// A logical
+/// </summary>
+/// <param name="LoadoutId"></param>
+public record struct LoadoutCursor(LoadoutId LoadoutId);
```

Comment:
Don't forget the docs here :wink:

Reply:
It has docs...just not a full sentence :P  . And actually that file should be deleted, not sure why it got left in the PR.